ccdproc/tests/test_image_collection.py

```diff
@@ -101,6 +101,13 @@ def test_filenames_are_set_properly(self, triage_setup):
             location=triage_setup.test_dir, filenames=fn, keywords=['filter'])
         assert img_collection.files == [fn]
 
+    def test_keywords_deleter(self, triage_setup):
+        ic = image_collection.ImageFileCollection(triage_setup.test_dir,
+                                                  keywords='*')
+
+        del ic.keywords
```

Comment:
to make the check more robust please also check that the `keywords` before deleting them are not empty.